### PR TITLE
Order Waitinglist by creation date

### PR DESCRIPTION
**Pull request by realmayus**
Closes https://github.com/corona-school/project-user/issues/647.
This PR introduces a new entity, `waiting_list_enrollment`, which stores the enrollment date of a pupil alonside the pupil's ID.
~~(This isn't quite ready for review yet)~~
@Jonasdoubleyou How could I test the DB migration? Whenever I restart my server, the database is reset, so I don't have any data to test it with. 🤔

### CODE REVIEW

File: web/dev.ts [hunk 1]
```diff
@@ -1019,6 +1032,9 @@ export async function setupDevDB() {
         await entityManager.save(Subcourse, subcourse);
         console.log('Inserted SubCourse.');
     }
+    // await entityManager.save(WaitingListEnrollment, waitingListEnrollment1);
+    // await entityManager.save(WaitingListEnrollment, waitingListEnrollment2);
+    // console.log('Inserted WaitingListEnrollments.');
```
Comment (Jonasdoubleyou) [inline web/dev.ts:1037]: why commented out?

File: ormconfig.js [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
-        //the output directories for the CLI are different from the directories where the builded files are
-        entitiesDir: "./common/entity",
-        migrationsDir: "./common/migration",
-        subscribersDir: "./common/subscriber",
-    },
-};
+module.exports = {
+    name: 'default',
+    type: 'postgres',
+    url: 'postgres://postgres@localhost:5432/coronaschool-dev',
```
Comment (Jonasdoubleyou) [inline ormconfig.js:4]: ...

File: common/courses/participants.ts [hunk 3]
```diff
@@ -73,10 +73,16 @@ export async function joinSubcourseWaitinglist(subcourse: Subcourse, pupil: Pupi
...
+    if (await isOnWaitingList(subcourse, pupil)) {
+        throw new RedundantError('Pupil is already on waiting list');
+    }
 
     try {
-        await prisma.subcourse_waiting_list_pupil.create({
-            data: { pupilId: pupil.id, subcourseId: subcourse.id },
+        await prisma.waiting_list_enrollment.create({
+            data: {
+                subcourseId: subcourse.id,
```
Comment (Jonasdoubleyou) [inline common/courses/participants.ts:83]: Does Prisma set createdAt?

**Review by Jonasdoubleyou — APPROVED** [review]

File: common/courses/participants.ts (hunk 3 above)
Comment (realmayus) [reply common/courses/participants.ts:83]: Yes, it should: https://github.com/corona-school/backend/blob/44cabf280ed7b8c0d9dfff7611e05dbfbbbd5b77/prisma/schema.prisma#L659

**Pull request merged**